crates/bevy_sprite/src/sprite.rs

```diff
@@ -162,6 +162,9 @@ pub enum SpriteImageMode {
     /// The sprite will take on the size of the image by default, and will be stretched or shrunk if [`Sprite::custom_size`] is set.
     #[default]
     Auto,
+    /// The texture will be scaled to fit the rect bounds defined in [`Sprite::custom_size`].
+    /// Otherwise no scaling will be applied.
+    ScaleMode(TextureScale),
```

Comment:
We shouldn't duplicate `Mode` here. Just `SpriteImageMode::Scale` is clearer here.